main.c

```diff
@@ -399,10 +425,14 @@ STATIC bool run_code_py(safe_mode_t safe_mode, bool *simulate_reset) {
         };
         #endif
 
+        supervisor_allocation *pystack;
```

Comment:
You could have
```
supervisor_allocation *pystack = NULL;
```
and avoid the `#else` branch.

Reply:
Thank you very much!